roles/deploy_bmh/template/bmh.yml.j2

```diff
@@ -20,9 +20,8 @@ spec:
   bootMACAddress: {{ nic.mac }}
 {% endfor                                                                            %}
   bootMode: {{ node_data['boot_mode'] }}
-  online:
-{%- if node_data['status'] == "running"                                              %}
- true
-{% else                                                                              %}
- false
+  online: {{ 'true' if node_data['status'] == "running" else 'false' }}
```

Comment:
<3

Reply:
ah, you're using that data from the vbmc output? Good to know it's useful :). We'll need the same from sushy I guess?

Reply:
@abays Good to know, I'll amend this by allowing `status` to not be passed and simply default it to false. For real BM we won't pass the value and that will be enough.